impress-org/givewp
# Remove login form after successful login in donation form with multi step form template

Resolves #5683 

Created few issues when testing this pull request:

- https://github.com/impress-org/givewp/issues/5690
- https://github.com/impress-org/givewp/issues/5691

## Description

In multi-step form template donor personal information section has a different location that means this section does not auto-update when the payment gateway reload. We do it manually by `$( document ).on( 'give_gateway_loaded', refreshPersonalInformationSection );`.  Previously this action was limit to a number of payment gateway count. I removed the limitation and from now this section will update on every payment gateway reload which prevent unnecessary UI glitch.

## Affects

it will not affect existing functionality and this change limited to donation form `Registration` type.
I renamed a function (internal use only) and call the form field setup function (`setupRegistrationFormInputFields`, `setupFFMInputs` etc. ) independently to set up form fields correctely.

## Testing Instructions

- Test this pull request with a different combination of `Guest Donations` and `Registration` donation forms. Donors must see the correct personal information, registration, and login form field. The field must be correct after the donor login on the donation form.
- Also make sure you also test this pull request with one and more than one activate payment gateways.

![image](https://user-images.githubusercontent.com/1784821/110582540-5ec35500-8192-11eb-94f6-d4b029b82b84.png)


## Pre-review Checklist

-   [x] Acceptance criteria satisfied and marked in the related issue
